pkgs/development/beam-modules/erlang-ls/rebar-deps.nix

```diff
@@ -79,7 +79,7 @@ let
         owner = "erlang-ls";
         repo = "yamerl";
         rev = "9a9f7a2e84554992f2e8e08a8060bfe97776a5b7";
-        sha256 = "1gb44v27paxwxm443m5f554wiziqi2kd300hgjjdg6fyvy3mvhss";
```

Comment:
This comes from a generator, one that I did not previously include.

https://github.com/NixOS/nixpkgs/blob/98f41c4a46642a095d14b7b1eaccead52ade740b/pkgs/development/beam-modules/erlang-ls/default.nix#L88

https://github.com/erlang-nix/rebar3_nix/blob/a9c8a6948bb88a718dbc7b62b9674b558e566633/src/rebar3_nix_lock_prv.erl#L27